packages/graphback-core/tests/subscriptionFilterTest.ts

```diff
@@ -0,0 +1,736 @@
+/* eslint-disable max-lines */
+import { Scalars, Maybe, QueryFilter, StringInput, IdInput, BooleanInput, IntInput, FloatInput, DateTimeInput } from '../src/runtime/QueryFilter';
+import { createSubscriptionFilterPredicate } from '../src/runtime/subscriptionFilter';
+
```

Comment:
General comment: I love these tests! Thank you for taking the time to add them. 

Can we remove `subscription`? Same comment as the one I made above for the `file under test`.